cmd/prometheus/main_test.go

```diff
@@ -26,12 +26,13 @@ import (
 	"time"
 
 	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+
```

Comment:
Why are you adding a blank line, here and elsewhere?

Reply:
Please do, prometheus/prometheus is what's local here.

Reply:
The last commit refactors the whole repo imports, with prometheus/prometheus as local.